### PR TITLE
added debuging cluster setup in developer documentation

### PR DESCRIPTION
**Pull request by xinredhat**
* Add information for login to cluster
* Add information for debugging cluster setup

### CODE REVIEW

**Comment by Roming22** [comment]
@xinredhat Please process the comments so we can merge the PR.

File: ci/docs/continuous_integation.md [hunk 1]
```diff
@@ -29,3 +29,15 @@ This is an issue when troubleshooting is required.
 
 To bypass deletion of the test cluster, delete the `destroy-cluster.txt` file in created in the root of the cloned repository.
 There is a 15 minutes window to log onto the container running the `destroy-cluster` step and delete the file.
+
+
+### Login to cluster
+When the test cluster is remained, to access the test cluster, you can go to the task named `deploy-cluster` to find the content of kubeconfig and login username/password.
```
[the remark this answers is not in the document]
Comment (xinredhat) [reply ci/docs/continuous_integation.md:35]: When someone wants to login to console, the `username/password` is easy way to do that

File: ci/docs/continuous_integation.md [hunk 2]
```diff
@@ -29,3 +29,15 @@ This is an issue when troubleshooting is required.
 
 To bypass deletion of the test cluster, delete the `destroy-cluster.txt` file in created in the root of the cloned repository.
 There is a 15 minutes window to log onto the container running the `destroy-cluster` step and delete the file.
+
+
+### Login to cluster
+When the test cluster is remained, to access the test cluster, you can go to the task named `deploy-cluster` to find the content of kubeconfig and login username/password.  
+
+### Debugging cluster setup
+When the CI failed at Pipeline-service Setup, you can ssh to the ci-runner pod in the test clsuter to manually execute `dev_setup.sh` for debugging cluster setup. for example
```
[the remark this answers is not in the document]
Comment (xinredhat) [reply ci/docs/continuous_integation.md]: fixed

File: ci/docs/continuous_integation.md [hunk 3]
```diff
@@ -29,3 +29,15 @@ This is an issue when troubleshooting is required.
 
 To bypass deletion of the test cluster, delete the `destroy-cluster.txt` file in created in the root of the cloned repository.
 There is a 15 minutes window to log onto the container running the `destroy-cluster` step and delete the file.
+
+
+### Login to cluster
+When the test cluster is remained, to access the test cluster, you can go to the task named `deploy-cluster` to find the content of kubeconfig and login username/password.  
+
+### Debugging cluster setup
```
[the remark this answers is not in the document]
Comment (xinredhat) [reply ci/docs/continuous_integation.md]: fixed

File: ci/docs/continuous_integation.md [hunk 4]
```diff
@@ -29,3 +29,15 @@ This is an issue when troubleshooting is required.
 
 To bypass deletion of the test cluster, delete the `destroy-cluster.txt` file in created in the root of the cloned repository.
 There is a 15 minutes window to log onto the container running the `destroy-cluster` step and delete the file.
+
+
+### Login to cluster
```
[the remark this answers is not in the document]
Comment (xinredhat) [reply ci/docs/continuous_integation.md]: fixed

**Review by Roming22 — APPROVED** [review]

**Pull request merged**